wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Commands.java

```diff
@@ -223,6 +223,22 @@ public static Command sequence(Command... commands) {
     return new SequentialCommandGroup(commands);
   }
 
+  /**
+   * Runs individual commands in a series without grouped sequence behavior.
+   *
+   * <p>Each command is run independently by proxy. The requirements of
+   * each command are reserved only for the duration of that command and
+   * are not reserved for an entire group process as they are in a
+   * grouped sequence.
+   *
+   * @param commands the commands to include in the series
+   * @return the command to run the series of commands
+   * @see #sequence() use sequence() to invoke group sequence behavior
```

Comment:
```suggestion
   * @see #sequence(Command...) use sequence() to invoke group sequence behavior
```
The parameter type(s) of a method must be specified. (See https://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html#see)